app/models/user.rb

```diff
@@ -26,6 +26,7 @@ module Permissions
     FORCE_PUBLISH_ANYTHING = "Force publish anything".freeze
     GDS_ADMIN = "GDS Admin".freeze
     EXPORT_DATA = "Export data".freeze
+    CAN_SHARE_PREVIEWS = "can share previews".freeze
```

Comment:
Let's ask someone with access to add this to production. Maybe we should also capitalise it for consistency? (We can remove the non capitalised one from integration and add it again.